backend/docker-compose.yml

```diff
@@ -1,3 +1,6 @@
+# Easy startup of Tracim backend services needed for testing with pytest
```

Comment:
Is it possible to change naming of this docker-compose.yml file, to add explicit information about its used to test tracim ?